lib/response_bank/response_cache_handler.rb

```diff
@@ -145,6 +145,11 @@ def serve_from_cache(cache_key_hash, match_entity_tag = "*", cache_age_tolerance
         # regen
         @headers.merge!(headers)
 
+        # If @headers['Content-Encoding'] is nil, just treat it as a cache miss
```

Comment:
You don't need this comment, and we might want to add a log in this case as this seems like an error state that we should make aware to users of the gem.

Reply:
Addressed